tests/test_indexer.py

```diff
@@ -268,11 +268,60 @@ def test_subscription_notification_unsuccessful(self):
                          f"WARNING:.*:Failed notification for subscription {subscription_id}"
                          f" for bundle {bundle_id} with transaction id .+ Code: {error_response_code}")
 
-    def test_alias_exists(self):
+    def test_get_index_shape_identifier(self):
```

Comment:
Don't test different conditions in the same test. Either create a new test case or use subtest.

Reply:
resolved